java/cdk-custom-resources/src/main/java/sleeper/cdk/custom/TableDefinerLambda.java

```diff
@@ -96,9 +97,17 @@ public void handleEvent(CloudFormationCustomResourceEvent event, Context context
     private void addTable(TableProperties tableProperties, TablePropertiesStore tablePropertiesStore,
             Map<String, Object> resourceProperties) throws IOException {
         String tableName = tableProperties.get(TABLE_NAME);
+
         LOGGER.info("Validating table properties for table {}", tableName);
         tableProperties.validate();
 
+        //Table may just be offline from a previous delete call
+        if (tablePropertiesStore.doesTableExistByName(tableName)) {
+            LOGGER.info("Table {} already exists. Updating it's properties", tableName);
+            tablePropertiesStore.save(tableProperties);
+            return;
+        }
```

Comment:
I think by default we should probably still fail the deployment if a table already exists, since you might do this accidentally.

From the issue:

> We could add an instance and table property to let you import an existing table into the CDK, which would change the way the add table operation works in the custom resource lambda.